openstack/nova/alerts/openstack/vmware-rebalancer.alerts

```diff
@@ -0,0 +1,16 @@
+groups:
+  - name: vmware-rebalancer.alerts
+    rules:
+      - alert: VcenterLowNumberOfPlaceableVms
+        expr: wm_vmware_rebalancer_placeable_vms_per_vcenter{has_bbs_with_custom_hana_exclusive_host="False",has_bbs_without_custom_hana_exclusive_host="True",flavor="g_c128_m512"} <= 3
```

Comment:
`has_bbs_with_custom_hana_exclusive_host="False"` excludes vCenters that should not be excluded (false negatives).